### ISSUE TITLE
[Bug]: Stirling Council (UK) API No Longer Working

### ISSUE THREAD

**Issue by belnarlo** [mampfes/hacs_waste_collection_schedule#2761]
### I Have A Problem With:

A specific source

### What's Your Problem

After the latest update, I had to recreate my integration. After stepping through the steps and hitting "OK" it returns:

The source returned an invalid response: "'NoneType' object has no attribute 'text'". Please check the provided arguments and try again.

As part of looking at it I reverted to the previous version of the integration which has the same problem so I have to assume the council changed their API and broke the integration :(

### Source (if relevant)

stirling_uk

### Logs

```Shell
This error originated from a custom integration.

Logger: custom_components.waste_collection_schedule.waste_collection_schedule.source_shell
Source: custom_components/waste_collection_schedule/waste_collection_schedule/source_shell.py:147
integration: Waste Collection Schedule (documentation)
First occurred: 13:54:05 (1 occurrences)
Last logged: 13:54:05

fetch failed for source Stirling.gov.uk: Traceback (most recent call last): File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source_shell.py", line 145, in fetch entries = self._source.fetch() ^^^^^^^^^^^^^^^^^^^^ File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source/stirling_uk.py", line 70, in fetch NextCollection.text.strip().split(REM_STRING1, 1)[0].strip(), ^^^^^^^^^^^^^^^^^^^ AttributeError: 'NoneType' object has no attribute 'text'
```

### Relevant Configuration

```YAML
Not sure the "route" code matters, but I hit the issue using route 9779
```

### Checklist Source Error

- [ ] Use the example parameters for your source (often available in the documentation) (don't forget to restart Home Assistant after changing the configuration)
- [X] Checked that the website of your service provider is still working
- [ ] Tested my attributes on the service provider website (if possible)
- [X] I have tested with the latest version of the integration (master) (for HACS in the 3 dot menu of the integration click on "Redownload" and choose master as version)

### Checklist Sensor Error

- [ ] Checked in the Home Assistant Calendar tab if the event names match the types names (if types argument is used)

### Required

- [X] I have searched past (closed AND opened) issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by 5ila5** [comment]
fixed by #2764 

This will be part of the next release or is available now as master (git) version 

install master/git: Developer tools -> actions -> select `update: Install update` -> click `Coose entity` and select WCS -> select `version` and type `master` in the input field -> run the action -> restart HA


<details><summary>Older versions of HACS:</summary>
<p>
install master with HACS version < 2.0: (3-dot menu -> redownload -> change version to master -> fully restart HA (this will increase update cycle by a lot)
</p>
</details>

**Comment by belnarlo** [comment]
Thanx for the quick response. Tested and it works perfectly again on my instance.
